Packs/Cryptosim/IncidentFields/incidentfield-Cryptosim_Correlation_ID.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
@haldunerbay please add a prefix to the cliname, id and name of the field. Also, please associate it with the cryptosim incident type and not with all of the types.

Reply:
do you mean prefix is cryptosim for fields ?

Reply:
Yes, you should add "cryptosim" as the prefix. For example, here:
cliname: "cryptosimcorrelationid"
id: "incident_cryptosimcorrelationid"
name: "Cryptosim Correlation ID"

Reply:
Added.